src/components/Login/Logout.jsx

```diff
@@ -9,45 +21,59 @@ import { useSession, LogoutButton } from '@inrupt/solid-ui-react';
  * @name Logout
  */
 
+
+
 const Logout = () => {
+
   const { session } = useSession();
-  const [showConfirmation, setShowConfirmation] = useState(false);
   localStorage.setItem('loggedIn', true);
+  // state for LogoutModal popup
+  const [showConfirmation, setShowConfirmation] = useState(false);
 
-  // Event handler for logging out of PASS and removing items from localStorage
+  // Event handler for logging out of SOLID POD and removing items from localStorage
```

Comment:
Yeah, I think this should be the case with Community Solid Server and we're finally logging out of the Solid Pod as well as PASS.

So I think this is fine, just need to confirm it on my part.